<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>451</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Background: rgb(234, 230, 224);
border-radius: 5px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="NameInput">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #FFFFFF;
color: rgb(28, 28, 28);
border: 1.5px solid #535366;
border-radius: 5px; </string>
     </property>
     <property name="placeholderText">
      <string>이름을 입력하세요.</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PasswdInput">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>230</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #FFFFFF;
color: rgb(28, 28, 28);
border: 1.5px solid #535366;
border-radius: 5px; </string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>비밀번호를 입력하세요.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LoginButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>330</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { 
    background-color: #535366; 
    color: white; 
    border-radius: 5px; 
    }
 QPushButton:hover { 
    background-color: #555; 
    }
  QPushButton:pressed { 
    background-color: #777; 
    }</string>
     </property>
     <property name="text">
      <string>로그인</string>
     </property>
    </widget>
    <widget class="ClickableLabel" name="RegisterButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>290</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: #535366; /* 파란색 텍스트 */
    font-weight: bold;
    text-decoration: underline; /* 밑줄 추가 */
}
QLabel:hover {
    color: #4682B4; /* 마우스를 올렸을 때 색상 변경 */
}</string>
     </property>
     <property name="text">
      <string>회원가입</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="IpInput">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #FFFFFF;
color: rgb(28, 28, 28);
border: 1.5px solid #535366;
border-radius: 5px; </string>
     </property>
     <property name="placeholderText">
      <string>IP를 입력하세요.</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PortInput">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #FFFFFF;
color: rgb(28, 28, 28);
border: 1.5px solid #535366;
border-radius: 5px; </string>
     </property>
     <property name="placeholderText">
      <string>Port번호를 입력하세요.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CloseButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>330</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { 
    background-color: #535366; 
    color: white; 
    border-radius: 5px; 
    }
 QPushButton:hover { 
    background-color: #555; 
    }
  QPushButton:pressed { 
    background-color: #777; 
    }</string>
     </property>
     <property name="text">
      <string>닫기</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header location="global">clickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
